app-services/secrets/res/configuration.toml

```diff
@@ -52,17 +51,35 @@ Host = "localhost"
 Port = 27017
 Timeout = "30s"
 
+# Shared Secret Store
 [SecretStore]
   Host = 'localhost'
   Port = 8200
-  Path = '/v1/secret/edgex/appservice'
+  Path = '/v1/secret/edgex/edgex-appservice/'
   Protocol = 'http'
-  Tokenfile = 'res/token.json'
+  TokenFile = './res/token.json'
+  # Number of attempts to retry retrieving secrets before failing to start the service.
+  AdditionalRetryAttempts = 10
+  # Amount of time to wait before attempting another retry
+  RetryWaitPeriod = "1s"
 
   [SecretStore.Authentication]
-  AuthType = 'X-Vault-Token'
-  # Vault root token that's used for authentication
-  AuthToken = 's.imTIVMl1OWhlX3zMAhSyaFlO'
+    AuthType = 'X-Vault-Token'
+
+# Exclusive Secret Store
+[SecretStoreExclusive]
+  Host = 'localhost'
+  Port = 8200
+  Path = '/v1/secret/edgex/secrets-example/'
+  Protocol = 'http'
+  TokenFile = './res/token.json'
```

Comment:
Missing `  RootCaCertPath = '/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem' `

Reply:
should it be `/tmp/edgex/secrets/ca/ca.pem`? the compose file in developer scripts have SecretStore_RootCaCertPath: /tmp/edgex/secrets/ca/ca.pem in the common env variables

Reply:
That is for Docker. This is the value used in all the TOML files for non-docker.

Reply:
'/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem' doesn't exist when I'm running the service natively, without edgex.
where would it be coming from?

Reply:
No clue, never run it natively. Check with Jim Wang. This is how all the other services are configured, so for consistency this should also.

Reply:
should be in the `DeployDir` specified in the toml file like this example from `security-secrets-setup`: https://github.com/edgexfoundry/edgex-go/blob/master/cmd/security-secrets-setup/res/configuration.toml

Reply:
@lenny-intel, the `RootCaCertPath = '/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem'` configuration is legacy. The link Jim posted above shows that the cert gets deployed to /tmp/edgex/secrets.

For my example, I don't need to configure the root cert path because i am running dev server for vault and not using TLS.